File: C++/C++Basic/06.Conditional-Statements-Advanced-Exercise/01.cinema.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    string typeProject;
    int row, col;

    cin >> typeProject;
    cin >> row >> col;

    int allSeats = row * col;

    double priceTickets = 0;

    if (typeProject == "Premiere")
    {
        priceTickets = 12.0;
    }
    else if (typeProject == "Normal")
    {
        priceTickets = 7.50;
    }

    else if (typeProject == "Discount")
    {
        priceTickets = 5.0;
    }

    priceTickets *= allSeats;

    cout.setf(ios::fixed);
    cout.precision(2);

    cout << priceTickets << " leva" << endl;

    return 0;

}

File: Python/Basic/06.Conditional-Statements-Advanced-Exercise/01.cinema.py
projection_type = input()
row = int(input())
col = int(input())

totalSpace = row * col

price = 0

if projection_type == "Premiere":
    price = totalSpace * 12
elif projection_type == "Normal":
    price = totalSpace * 7.50
else:
    price = totalSpace * 5.0

print(f"{price:.2f} leva")
